test/integration_test/process/run.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -u
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $cur
+
+bin_path=../../../bin
+
+${bin_path}/dummy > dummy.out &
+
+sleep 1
+
+pid=$(cat dummy.out)
+
+# test stop attack
+${bin_path}/chaosd attack process stop -p ${pid} > proc.out
+
+sleep 1
+
+uid=$(cat proc.out | grep "Attack process ${pid} successfully" | awk -F: '{print $2}')
+
+stat=$(ps axo pid,s | grep ${pid} | awk '{print $2}')
+
+if [[ ${stat} != "T" && ${stat} != "t" ]]; then
```

Comment:
how about adding a comment of what the `t` means?

Reply:
Fixed